pypeit/par/pypeitpar.py

```diff
@@ -1164,6 +1164,13 @@ def __init__(self, ex_value=None, flux_value=None, nmaskedge=None,
         dtypes['coaddfile'] = str
         descr['coaddfile'] = 'Output filename'
 
+
+        # Version checking
+        defaults['chk_version'] = True
+        dtypes['chk_version'] = bool
+        descr['chk_version'] = 'If True enforce strict PypeIt version checking to ensure that spec1d*.fits files were created' \
+                               'with the current version of PypeIt'
+
```

Comment:
It isn't strictly the pypeit version that matters.  It's the version of the data model for the SpecObj object.  So how about instead:
```
descr['chk_version'] = 'If True, enforce strict PypeIt version checking.  Beware that setting this to False may cause other obscure faults or silent bugs.'
```
And a general note, when you break strings like this, make sure you add relevant spacing.  The two strings are directly combined meaning that, as you've written it, the resulting spring will include the word `createdwith`.